src/main/java/core/basesyntax/service/impl/ValidFirstLinePredicate.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import java.util.function.Predicate;
+
+public class ValidFirstLinePredicate implements Predicate<String> {
+    private static final String INVALID_FIRST_LINE_START = "type";
+
+    @Override
+    public boolean test(String element) {
+        return !element.isEmpty()
+                && element.trim().toLowerCase().startsWith(INVALID_FIRST_LINE_START);
```

Comment:
The logic in the `test` method seems to be inverted. The method currently returns true if the line starts with `INVALID_FIRST_LINE_START`, which implies it's invalid. Consider revising the logic to return false in such cases, as the method name suggests it should validate the line.